frontend/web/styles/components/_input.scss

```
@@ -33,19 +33,19 @@ textarea {
 .input-container.input-underline {
   input.input {
     border: none;
-    border-bottom: 1px solid $input-border-color;
+    border-bottom: 1px solid var(--color-border-default);
     border-radius: 0;
     background-color: transparent;
     padding-left: 8px;
     padding-right: 8px;
     // _forms.scss re-applies a full border shorthand on hover/focus.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)